en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,859 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title: Asgardeo - Action Management Rest API
+  description:
+    "This is the RESTful API for managing action configuration for an organization in WSO2 Asgardeo. \n
```

Comment:
why do we say "WSO2 Asgardeo" ? Let's just say as "Asgardeo"

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/ee4f5ca7c24d3d8dcaca4943b7b5aadb6e16e7fc